app/helpers/application_helper/toolbar/ems_physical_infras_center.rb

```diff
@@ -27,23 +27,23 @@ class ApplicationHelper::Toolbar::EmsPhysicalInfrasCenter < ApplicationHelper::T
         button(
           :ems_physical_infra_new,
           'pficon pficon-add-circle-o fa-lg',
-          t = N_('Add a New Infrastructure Provider'),
+          t = N_('Add a New Physical Infrastructure Provider'),
           t,
           :url => "/new"),
         button(
           :ems_physical_infra_edit,
           'pficon pficon-edit fa-lg',
-          N_('Select a single Infrastructure Provider to edit'),
-          N_('Edit Selected Infrastructure Providers'),
+          N_('Select a single Physical Infrastructure Provider to edit'),
+          N_('Edit Selected Physical Infrastructure Providers'),
           :url_parms    => "main_div",
           :send_checked => true,
           :enabled      => false,
           :onwhen       => "1"),
         button(
           :ems_physical_infra_delete,
           'pficon pficon-delete fa-lg',
-          N_('Remove selected Infrastructure Providers from Inventory'),
-          N_('Remove Infrastructure Providers from Inventory'),
+          N_('Remove selected Physical Infrastructure Providers from Inventory'),
+          N_('Remove Physical Infrastructure Providers from Inventory'),
           :url_parms    => "main_div",
           :send_checked => true,
           :confirm      => N_("Warning: The selected Infrastructure Providers and ALL of their components will be permanently removed!"),
```

Comment:
Also here? ;)

Reply:
Thank you @himdel , I also add in these lines